### PR TITLE
Feat: Implement Google calendar schedular.

### PR DESCRIPTION
**Pull request by Jamerrone**
## What changes were made
A "Person" can now have a "Meet". New `/meet/:personSlug` pages are created for every person with a meet. It's a simple page without a footer, its only content is a Google Calendar iframe.

## How to test or check results

Right now, `/meet/jasper` is the only existing meet page. You can get there from the bottom of the https://deploy-preview-780--voorhoede-website.netlify.app/en/impact/ page or directly: https://deploy-preview-780--voorhoede-website.netlify.app/en/meet/jasper/.

## Checks
- [X] All content model changes are done through [scripted migrations](../readme.md#scripted-migrations)
- [X] Appointed PR reviewers

### CODE REVIEW

File: src/composables/useDatoNuxtRoute.ts [hunk 1]
```diff
@@ -51,6 +52,9 @@ export function useDatoNuxtRoute(page: Page) {
     case 'PersonRecord': {
       return { name: 'language-team-slug', params: { ...sharedParams, slug: page.slug } }
     }
+    case 'MeetRecord': {
+      return { name: 'language-meet-slug', params: { ...sharedParams, slug: page.parents?.[0].slug } }
```
Comment (Siilwyn) [inline src/composables/useDatoNuxtRoute.ts:56]:
```suggestion
      return { name: 'language-meet-slug', params: { ...sharedParams, slug: page.parents[0].slug } }
```

Could this work? Think we'd want to hard fail on a missing parent?

File: src/layouts/content-page.vue [hunk 2]
```diff
@@ -42,14 +43,21 @@
...
   const skipLink = ref(null);
   const mobileMenuIsOpen = ref(false);
   const focusTrapMobileMenu = computed(() => mobileMenuIsOpen.value ? true : null);
 
+  defineProps({
+    hideFooter: {
+      type: Boolean,
+      default: false,
+    },
+  });
```
Comment (Siilwyn) [inline src/layouts/content-page.vue:59]: Let's use the TS native types like so: `defineProps<{}>()`

File: src/pages/[language]/events/index.vue [hunk 3]
```diff
@@ -55,6 +55,8 @@
 </template>
 
 <script setup>
+  definePageMeta({ layout: 'content-page' })
```
Comment (Siilwyn) [inline src/pages/[language]/events/index.vue:58]: If you don't mind too much could you find-and-replace all of these to add a semicolon? :sweat_smile: 
```suggestion
  definePageMeta({ layout: 'content-page' });
```

File: src/pages/[language]/meet/[slug]/index.query.graphql [hunk 4]
```diff
@@ -0,0 +1,24 @@
+query MeetSlug($slug: String, $locale: SiteLocale) {
+  person(filter: { slug: { eq: $slug }, meet: { exists: true } }, locale: $locale) {
```
Comment (Siilwyn) [inline src/pages/[language]/meet/[slug]/index.query.graphql:2]: `meet: { exists: true }` nice! TIL

**Review by Siilwyn — CHANGES_REQUESTED** [review]

File: src/layouts/content-page.vue (hunk 2 above)
Comment (Jamerrone) [reply src/layouts/content-page.vue:59]: As discussed, skipping this for now.

**Review by Siilwyn — APPROVED** [review]

**Pull request merged**